.github/workflows/srt.yml

```diff
@@ -39,7 +39,7 @@ jobs:
 
       - id: load-env
         run: |
-          sudo apt-get update
+          #sudo apt-get update
```

Comment:
Just making sure this is on purpose.

Reply:
@ekluzek Yes it's intentional we can remove the commented line if you prefer.

Reply:
I think removing the line is a better practice when we have the history in version control. The only reason to comment it out is if there are some conditions where you'd add it back in again. But, then it should have a comment about why you'd do that.